libnopegl/src/pgcraft.c

```diff
@@ -1167,7 +1161,9 @@ static int filter_pipeline_elems(struct pgcraft *s,
         if (info_map) {
             const size_t id = *(size_t *)desc_elem;
             const char *name = ngli_pgcraft_get_symbol_name(s, id);
-            const struct program_variable_info *info = ngli_hmap_get(info_map, name);
+            char resource_name[MAX_ID_LEN];
+            snprintf(resource_name, sizeof(resource_name), "%s%s", name, suffix);
```

Comment:
no need to keep the `len >= sizeof(name)` check?

Reply:
Re-added the check.

Reply:
Also added in program_gl_utils.c